home/views.py

```diff
@@ -31,8 +31,7 @@ def search_view(request, page: str = "1"):
         # Populated search scenario
         form = SearchForm(request.GET)
         if not form.is_valid():
-            print("form error on validation")
-            print(form.errors)
+            return HttpResponseBadRequest(form.errors)
```

Comment:
I don't think this is correct - if there are form errors we should still render the page, just with an error message at the top

the design system has this page: https://design-system.service.gov.uk/patterns/validation/

I think we just skipped it for now because we didn't implement any validation yet

Reply:
yeah, perhaps this bit is pointless. We need to do more error handling on the form before we can return any sensible content for the error page. However it's ok as a holding pattern, and the tests weren't working for a bad form unless I returned something here (it'd just hang)